.github/workflows/deploy_to_github_io.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           cd hyp3-docs
           # Remove conflicting CNAME for project site
           rm docs/CNAME
+          # Set site URL for dev site
+          sed -i 's/site_url: https:\/\/hyp3-docs.asf.alaska.edu\//site_url: https:\/\/hyp3-docs.asf.alaska.edu\/hyp3-docs\//' mkdocs.yml
```

Comment:
might be better using an alternate delimiter:
`'s|site_url: https://hyp3-docs.asf.alaska.edu/|site_url: https://hyp3-docs.asf.alaska.edu/hyp3-docs/|'

Reply:
ooh right, I always for get you can do that